spec/rtcHelpers.ts

```diff
@@ -26,8 +26,8 @@ export function createPublishMessage(channel: string, data: any, correlationId:
     type: RealTimeMessageTypes.Command,
     data: {
       command: RealTimeCommandTypes.Publish,
-      arguments: { channel, data },
-      correlation_id: correlationId
+      correlation_id: correlationId,
```

Comment:
Yes, there is. It's a bit tricky but I try to explain.
SDK uses request as a `Map` key and when API responds, it takes request from the responding object and seeking for it throughout the Map. The thing is that backend changed the order of properties in the response recently.
Here is the example I found:

that is what we used to send:
`{"type":"command","data":{"command":"publish","arguments":{"channel":"my_channel","data":"Voluptas tenetur aut."},"correlation_id":"0.7456339363372082"}}`

this is the response:
`{"type":"command","data":{"command":"publish","correlation_id":"0.7456339363372082","arguments":{"channel":"my_channel","data":"Voluptas tenetur aut."}}}`

Since that, there were two possible solutions: 
  1. refactoring all the RTC lib, use correlation id as a Map index (it also would need to add uuid library).
  2. just shift `correlation_id` a little bit :) 

I have chosen the second option :) If our API changes request structure again, we will need to get back to the first option.

Reply:
that was tricky. agreed then, just wanted to make sure it wasn't your IDE pranking you 😆